public/locales/en/messages.po

```diff
@@ -18,6 +18,166 @@ msgstr ""
 msgid "common.save"
 msgstr "Save"
 
+#: public/app/core/components/NavBar/NavBarItem.tsx
+msgid "nav.alerting"
+msgstr "Alerting"
+
+#: public/app/core/components/NavBar/NavBarItem.tsx
+msgid "nav.alerting-admin"
+msgstr "Admin"
+
+#: public/app/core/components/NavBar/NavBarItem.tsx
+msgid "nav.alerting-am-routes"
+msgstr "Notification policies"
+
+#: public/app/core/components/NavBar/NavBarItem.tsx
+msgid "nav.alerting-channels"
+msgstr "Notification channels"
+
+#: public/app/core/components/NavBar/NavBarItem.tsx
+msgid "nav.alerting-groups"
+msgstr "Groups"
+
+#: public/app/core/components/NavBar/NavBarItem.tsx
+msgid "nav.alerting-list"
+msgstr "Alert rules"
+
+#: public/app/core/components/NavBar/NavBarItem.tsx
+msgid "nav.alerting-receivers"
+msgstr "Contact points"
+
+#: public/app/core/components/NavBar/NavBarItem.tsx
+msgid "nav.alerting-silences"
+msgstr "Silences"
+
+#: public/app/core/components/NavBar/NavBarItem.tsx
+msgid "nav.api-keys"
+msgstr "API keys"
+
+#: public/app/core/components/NavBar/NavBarItem.tsx
+msgid "nav.change-password"
+msgstr "Change password"
+
+#: public/app/core/components/NavBar/NavBarItem.tsx
+msgid "nav.config"
+msgstr "Configuration"
+
+#: public/app/core/components/NavBar/NavBarItem.tsx
+msgid "nav.create"
+msgstr "Create"
+
+#: public/app/core/components/NavBar/NavBarItem.tsx
+msgid "nav.create-alert"
+msgstr "Akert rule"
```

Comment:
🤔 I thought these messages were extracted from the `defineMessage` ,  is this file missing a manual extract (`lingui extract)`?  


```suggestion
msgstr "Alert rule"
```

Reply:
Good catch!